src/figuresClasses.ts

```diff
@@ -1,11 +1,93 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | `circle` | `rectangle`;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  readonly shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
```suggestion
    public color: Figure['color']
```

Please, check other places as well